draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -16,21 +17,53 @@ const getRelativeParent = (element) => {
 };
 
 export default class Toolbar extends React.Component {
-
-  state = {
-    isVisible: false,
-    position: undefined,
-
+  static propTypes = {
     /**
-     * If this is set, the toolbar will render this instead of the regular
-     * structure and will also be shown when the editor loses focus.
-     * @type {Component}
+     * An indents from the left / right window borders
      */
-    overrideContent: undefined
+    toolbarMargin: PropTypes.number
+  };
+
+  static defaultProps = {
+    toolbarMargin: 5
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isVisible: false,
+      position: undefined,
+      /**
+       * If this is set, the toolbar will render this instead of the regular
+       * structure and will also be shown when the editor loses focus.
+       * @type {Component}
+       */
+      overrideContent: undefined,
+      /**
+       * We are holding default toolbar width to prevent geometry changing, that
+       * happens very often
+       */
+      width: null,
+      /**
+       * this is an additional unique toolbar class that will be used to
+       * manipulate css arrow position
+       * @type {string}
+       */
+      pointerClassName: '',
+      /**
+       * pointerClassName internals here. It could look like: "{ left: 1px; }"
+       * @type {number|string}
+       */
+      pointerShift: 0
+    };
+
+    // we need to flush toolbar styles on each appearance (especially its width)
+    this.toolbar = null;
```

Comment:
this.toolbar is undefined when this line is executed since the constructor is run before the component is mounted so I'm not sure what this line does?